packages/amplify-prompts/src/printer.ts

```diff
@@ -23,15 +23,15 @@ export class AmplifyPrinter implements Printer {
   };
 
   success = (line: string): void => {
-    this.writeSilenceableLine(`✅ ${chalk.green(line)}`);
+    this.writeSilenceableLine(`${!isHeadless ? '✅' : ''} ${chalk.green(line)}`);
```

Comment:
super-duper nit: this will add an extra space to the front of the line when headless is true. Also, I'd suggest flipping the ternary condition:
```suggestion
    this.writeSilenceableLine(`${isHeadless ? '' : '✅ '}${chalk.green(line)}`);
```
will fix it